cf/data/data.py

```diff
@@ -8811,32 +8812,27 @@ def equals(
         other_Units = other.Units
         if self_Units != other_Units:
             logger.info(
-                "{}: Different Units ({!r}, {!r}".format(
+                "{}: Different Units ({!r}, {!r})".format(
```

Comment:
I was planning to do the f-string conversions separately, as I am doing with the "Incremental conversion to f-strings: ..." daily commits on the `master` branch, to keep the functional changes separate as I thought it might make reviewing of the migration proper somewhat easier, but if you prefer those to be bundled in with each migration PR, that's fine by me. I guess if I do them as a separate commit for each PR, that gives us the best of both worlds, so that'll be my plan going forwards.

Reply:
OK, that is good - Feel free to adopt whatever approach to f-strings you like!